lib/course.rb

```diff
@@ -1 +1,21 @@
+class Course
+  attr_reader :name, :capacity, :students
 
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+  end
+
+  def full?
+    @students.length >= @capacity
+  end
+
+  def enroll(student)
+    if full?
+      'No more students can enroll'
+    else @students << student
+    end
```

Comment:
```suggestion
    else
      @students << student
    end
```

Just some formatting convention, the code we want to execute in the case of `else` should go on the line after `else`.